rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -474,16 +544,92 @@ void gs_frame::take_screenshot(const std::vector<u8> sshot_data, const u32 sshot
 				}
 			}
 
+			screenshot_manager manager;
```

Comment:
```suggestion
			screenshot_manager& manager = *g_fxo->get<screenshot_manager>();
```

Reply:
I'll probably refactor it into some utility later or tomorrow. Not gonna have time for a while